.pipelines/iis.servicemonitor.build.dev.yml

```diff
@@ -1,12 +1,31 @@
-name: $(Rev:rr)
+name: $(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.rr)
+
+pr:
+  - master
+
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+
+resources:
+  repositories:
+    - repository: MicrosoftIISCommon
+      type: github
+      name: Microsoft/IIS.Common
+      endpoint: GitHub-IIS-PAT
 
 jobs:
-- template: templates\build.yml
+- template: .azure\templates\build.yml@MicrosoftIISCommon
   parameters:
-    agentPoolName: 'VSEng-MicroBuildV2CustomPool'
-    agentPoolDemandTeam: 'IISAdmin'
+    agentPoolName: 'VSEng-MicroBuildVS2017'
+    agentPoolDemandTeam: ''
     solution: '**\ServiceMonitor.sln'
     productMajor: 2
     productMinor: 0
     signType: 'test'
-    publishArtifactInstaller: 'false'
+    indexSourcesAndPublishSymbols: 'true'
```

Comment:
For dev build, do we still want this task?

Reply:
I can remove it if we don't want source symbol publishing for dev builds. If anyone is installing dev builds, it might make debugging harder.

Reply:
Just to clarify: without this task, can we still download the symbols in the build artifact and use them for source debugging -  assuming the dev machine has the source code. Or is it for using the symbols on a machine without the source code?

Reply:
You can download the symbols as a build artifact but they won't automatically be loaded from the symbol server by the debugger. In addition the pdb's will not be source indexed i.e. the debugger will not automatically be able to load the source file. You would need to manually figure out the commit corresponding to the build, download it and then use that for your debugging.